packages/core/test/stylable-transformer/general.spec.ts

```diff
@@ -77,8 +77,22 @@ describe('Stylable postcss transform (General)', () => {
     });
 
     describe('experimentalSelectorInference', () => {
+        it('should set default inferred selector context to universal selector', () => {
+            testStylableCore(
+                `
+                    .root { -st-states: state; }
+                    .class { -st-states: state; }
+                
+                    /* @rule(unknown state) :state */
```

Comment:
There errors emitted here let's test them. I don't see any other place that test them.